bin/apt-dependencies.sh

```diff
@@ -172,6 +171,9 @@ if [ "$1" != "nojs" ]; then
   if [ "${VERSION_CODENAME}" == "bullseye" ]; then
     apt-get install --no-install-recommends -y libmozjs-78-dev
   fi
+  if [ "${VERSION_CODENAME}" == "bookworm" ]; then
+      apt-get install --no-install-recommends -y libmozjs-78-dev libmozjs-102-dev
```

Comment:
Why to add `libmozjs-102-dev` (SpiderMonkey 102)?  I do not think it is supported.

Reply:
I have only provided it in preparation for possible use in the image.

Reply:
All right.